include/psa/crypto.h

```diff
@@ -2245,6 +2310,18 @@ typedef struct {
  *                            specifying the public exponent. The
  *                            default public exponent used when \p extra
  *                            is \c NULL is 65537.
+ *                          - For an DSA key (\p type is
+ *                            #PSA_KEY_TYPE_DSA_KEYPAIR), \p extra is an
+ *                            optional structure specifying the key domain
```

Comment:
The one specifying the key domain parameters and whose documentation is referenced a bit later in the entry. Do you have a suggestion to make it more clear or less confusing?

Reply:
It's not a matter of clarity, it's a matter of completeness. You never specify what the structure is. This should probably not be a structure since `psa_set_key_domain_parameters` doesn't take a structure as input: it should be a buffer.

Reply:
It's an issue with clarity, because I didn't mean structure in the C struct case. I meant in the abstract sense, so I can clarify by saying "buffer" instead of "structure".